i18n/data.json

```diff
@@ -0,0 +1,26 @@
+{
+    "en": {
+        "login.username": "Nickname",
+        "login.joinRoom": "Join room",
+        "login.createRoom": "Create room",
+        "message.newRule": "New rule:",
```

Comment:
Remove colons from all translations. They are not language-specific information, and actually place restrictions on how they should be used in frontend.